templates/taxi/driver_list.html

```diff
@@ -1,34 +1,53 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
     <h1>
-      Driver List
-      <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
-        Create
-      </a>
-    </h1>
+    Driver List
+    <a href="{% url 'taxi:driver-create' %}" class="btn btn-primary link-to-page">
+      Create
+    </a>
+  </h1>
 
-    {% if driver_list %}
+  <form method="get" action="" class="form-inline">
+    {{ search_form|crispy }}
+    <input type="submit" class="btn" style="border: 1px solid gray;" value="🔍">
+  </form>
+
+  {% if driver_list %}
     <table class="table">
       <tr>
         <th>ID</th>
         <th>Username</th>
         <th>First name</th>
         <th>Last name</th>
         <th>License number</th>
+        <th>Update</th>
+        <th>Delete</th>
       </tr>
     {% for driver in driver_list %}
-      <tr>
-        <td>{{ driver.id }}</td>
-        <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a></td>
-        <td>{{ driver.first_name }}</td>
-        <td>{{ driver.last_name }}</td>
-        <td>{{ driver.license_number }}</td>
-      </tr>
+        <tr>
+          <td>{{ driver.id }}</td>
+          <td><a href="{{ driver.get_absolute_url }}">{{ driver.username }} {% if user == driver %} (Me){% endif %}</a>
+          </td>
+          <td>{{ driver.first_name }}</td>
+          <td>{{ driver.last_name }}</td>
+          <td>{{ driver.license_number }}</td>
+          <td>
+            <a href="{% url 'taxi:driver-update' pk=driver.id %}">
+              Update license number
```

Comment:
The link text 'Update license number' might be misleading if the update view allows updating more than just the license number. Consider changing it to a more general term like 'Update' if the view allows updating other fields as well.